registry/coder/modules/vscode-desktop/run.sh

```diff
@@ -28,14 +28,58 @@ check_vscode_cli() {
         "/Applications/Visual Studio Code.app/Contents/Resources/app/bin/code"
     )
     
-    for path in "$${vscode_paths[@]}"; do
+    for path in "${vscode_paths[@]}"; do
         if [ -x "$path" ]; then
             export PATH="$PATH:$(dirname "$path")"
             return 0
         fi
     done
+
+    # Try to install VS Code Desktop (which provides the CLI)
```

Comment:
<img width="906" height="400" alt="Image" src="https://github.com/user-attachments/assets/a6edcb8d-848e-4379-ba28-a2d732d984d4" />

I think we only need the CLI and not the full VS Code.